front-end/playground/src/PermanentComponents/Footer.js

```diff
@@ -0,0 +1,29 @@
+import React from "react"
+import Card from "react-bootstrap/Card";
+import './Footer.css';
+
+function Footer() {
+    return(
+        <div>
+
+
+            <Card bg="danger" className="footer">
+                <Card.Header  >Contact Information</Card.Header>
+                <Card.Body>
+                    <blockquote className="blockquote mb-0">
+                    <p>
```

Comment:
Flexbox provides "space-around" functionality

Reply:
Thanks I will try to take necessary actions.